lib/elixir_internal_certificate_web/controllers/api/v1/user_search_controller.ex

```diff
@@ -54,6 +55,43 @@ defmodule ElixirInternalCertificateWeb.Api.V1.UserSearchController do
     end
   end
 
+  def upload(
+        %{
+          private:
+            %{guardian_default_claims: %{"sub" => sub_user_id} = _guardian_default_claims} =
+              _private
+        } = conn,
+        %{"file" => file}
+      ) do
+    with {:ok, keywords} <- CsvParsingHelper.validate_and_parse_keyword_file(file),
+         {user_id, _} <- Integer.parse(sub_user_id, 10),
+         {_keywords_count, uploaded_keywords} <- Scrapers.create_user_search(keywords, user_id) do
+      conn
+      |> put_view(KeywordUploadView)
+      |> render("show.json", %{
+        data: uploaded_keywords
```

Comment:
Can we just return `:ok` or `:created` status here without the `uploaded_keywords` data @nvminhtue as on the API the client can call the `index` API back to get the list keyword